docs/templates/20-requirements.ftl

```diff
@@ -0,0 +1,10 @@
+# Requirements
+
+Requirements for ${solution_name} are:
+
+* Access to the internet
+* Jenkins versions 2.60.1 or higher. Non-LTS versions of Jenkins are not supported.
+* Java 8.
+* ${polaris_product_name}, and a valid access token.
+* The ${polaris_product_name} CLI must run successfully from the command line on the project you are building, and the polaris.yml
```

Comment:
While I'd say this is absolutely best practice, the Polaris CLI can run just fine without a preconfigured YAML, so long as the defaults that it uses when it generates the polaris.yml are acceptable to the user. I might change this requirement to: modified polaris.yml must be included in your source if needed to generate accurate results.